unit-tests/test-fw-update.py

```diff
@@ -13,22 +14,25 @@
 import pyrealsense2 as rs
 import pyrsutils as rsutils
 from rspy import devices, log, test, file, repo
-
-
-if not devices.acroname:
-    log.i( "No Acroname library found; skipping device FW update" )
-    sys.exit(0)
-# Following will throw if no acroname module is found
-from rspy import acroname
-try:
-    devices.acroname.discover()
-except acroname.NoneFoundError as e:
-    log.f( e )
-# Remove acroname -- we're likely running inside run-unit-tests in which case the
-# acroname hub is likely already connected-to from there and we'll get an error
-# thrown ('failed to connect to acroname (result=11)'). We do not need it -- just
-# needed to verify it is available above...
-devices.acroname = None
+import time
+
+# This is the first test running, discover acroname modules.
```

Comment:
Why do we care here if it's the first test that runs?

Reply:
This test runs first (priority 0) and is the only that calls `devices.acroname.discover()`.
I am testing based on `jetson` in context and not camera model, because it cannot be queried before this call.